mindjunkies/courses/tests/test_templatetags.py

```
@@ -0,0 +1,38 @@
+import pytest
+from django.template import Context, Template
+from mindjunkies.courses.templatetags import courses_tags
```

Ruff (0.8.2)

`mindjunkies.courses.templatetags.courses_tags` imported but unused

Remove unused import: `mindjunkies.courses.templatetags.courses_tags`

(F401)